### PR TITLE
Multipleexecutor

### PR DESCRIPTION
**Pull request by evlstyle**
fix the massive logging issue by leveling down the log level to debug, also fixed the typo in the servlet name.

### CODE REVIEW

File: azkaban-common/src/main/java/azkaban/executor/ExecutorManager.java [hunk 1]
```diff
@@ -241,7 +242,7 @@ private void refreshExecutors() {
             @Override
             public String call() throws Exception {
               return callExecutorForJsonString(executor.getHost(),
-                executor.getPort(), "/serverstastics", null);
+                executor.getPort(), "/serverstatistics", null);
```
Comment (logiclord) [inline azkaban-common/src/main/java/azkaban/executor/ExecutorManager.java]: camel case ?

File: azkaban-common/src/main/java/azkaban/executor/ExecutorManager.java [hunk 2]
```diff
@@ -1038,6 +1039,7 @@ public String submitExecutableFlow(ExecutableFlow exflow, String userId)
           Executor executor = activeExecutors.iterator().next();
           // assign only local executor we have
           reference.setExecutor(executor);
+          executorLoader.assignExecutor(executor.getId(), exflow.getExecutionId());
```
Comment (logiclord) [inline azkaban-common/src/main/java/azkaban/executor/ExecutorManager.java]: I have already covered this fix using dispatch method in https://github.com/azkaban/azkaban/pull/500/files

File: azkaban-common/src/main/java/azkaban/executor/ExecutorManager.java [hunk 3]
```diff
@@ -117,6 +117,7 @@
 
   public ExecutorManager(Props props, ExecutorLoader loader,
       Map<String, Alerter> alters) throws ExecutorManagerException {
+    alerters = alters;
```
Comment (logiclord) [inline azkaban-common/src/main/java/azkaban/executor/ExecutorManager.java]: already covered in #500

File: azkaban-common/src/main/java/azkaban/executor/ExecutorManager.java (hunk 1 above)
Comment (evlstyle) [inline azkaban-common/src/main/java/azkaban/executor/ExecutorManager.java]: fixed.

File: azkaban-common/src/main/java/azkaban/executor/ExecutorManager.java (hunk 2 above)
Comment (evlstyle) [inline azkaban-common/src/main/java/azkaban/executor/ExecutorManager.java]: removed.

File: azkaban-common/src/main/java/azkaban/executor/ExecutorManager.java (hunk 3 above)
Comment (evlstyle) [inline azkaban-common/src/main/java/azkaban/executor/ExecutorManager.java]: removed.

File: azkaban-common/src/main/java/azkaban/executor/ExecutorManager.java [hunk 4]
```diff
@@ -126,7 +126,7 @@ public ExecutorManager(Props props, ExecutorLoader loader,
         new QueuedExecutions(props.getLong(AZKABAN_WEBSERVER_QUEUE_SIZE, 100000));
     this.loadQueuedFlows();
 
-    alerters = alters;
```
Comment (logiclord) [inline azkaban-common/src/main/java/azkaban/executor/ExecutorManager.java]: You have a test failure because of this

**Comment by logiclord** [comment]
LGTM

File: azkaban-common/src/main/java/azkaban/executor/selector/CandidateComparator.java [hunk 5]
```diff
@@ -66,7 +66,7 @@ protected void registerFactorComparator(FactorComparator<T> comparator){
 
       // add or replace the Comparator.
       this.factorComparatorList.put(comparator.getFactorName(),comparator);
-      logger.info(String.format("Factor comparator added for '%s'. Weight = '%s'",
+      logger.debug(String.format("Factor comparator added for '%s'. Weight = '%s'",
```
Comment (hluu) [inline azkaban-common/src/main/java/azkaban/executor/selector/CandidateComparator.java]: do we want to wrap this in an if statement?

if (logger.isDebugLevel()) {
}

File: azkaban-common/src/main/java/azkaban/executor/selector/CandidateComparator.java [hunk 6]
```diff
@@ -104,7 +104,7 @@ public int getTotalWeight(){
    * @return a pair structure contains the score for both sides.
    * */
   public Pair<Integer,Integer> getComparisonScore(T object1, T object2){
-    logger.info(String.format("start comparing '%s' with '%s',  total weight = %s ",
+    logger.debug(String.format("start comparing '%s' with '%s',  total weight = %s ",
```
Comment (hluu) [inline azkaban-common/src/main/java/azkaban/executor/selector/CandidateComparator.java]: do we want to wrap this in an if statement?

if (logger.isDebugLevel()) {
}

File: azkaban-common/src/main/java/azkaban/executor/selector/CandidateFilter.java [hunk 7]
```diff
@@ -70,13 +70,13 @@ public boolean filterTarget(T filteringTarget, V referencingObject){
     boolean result = true;
     for (FactorFilter<T,V> filter : filterList){
       result &= filter.filterTarget(filteringTarget,referencingObject);
-      logger.info(String.format("[Factor: %s] filter result : %s ",
+      logger.debug(String.format("[Factor: %s] filter result : %s ",
```
Comment (hluu) [inline azkaban-common/src/main/java/azkaban/executor/selector/CandidateFilter.java]: do we want to wrap this in an if statement?

if (logger.isDebugLevel()) {
}

**Comment by johnyu0520** [comment]
LGTM

**Pull request merged**